extensions/NV/SPV_NV_displacement_micromap.asciidoc

```diff
@@ -0,0 +1,297 @@
+----
+
+
+
+New Capabilities
+----------------
+
+This extension introduces new capabilities:
+
+----
+DisplacementMicromapNV
+RayTracingDisplacementMicromapNV
+----
+
+
+New Builtins
+------------
+
+Builtins added under the *RayTracingDisplacementMicromapNV* capability
+
+----
+HitMicroTriangleVertexPositionsNV
+HitMicroTriangleVertexBarycentricsNV
+HitKindFrontFacingMicroTriangleNV
+HitKindBackFacingMicroTriangleNV
+----
+
+New Instructions
+----------------
+
+Instructions added under the *DisplacementMicromapNV* capability
+
+----
+OpFetchMicroTriangleVertexPositionNV
+OpFetchMicroTriangleVertexBarycentricNV
+----
+
+Modifications to the SPIR-V Specification, Version 1.1
+------------------------------------------------------
+
+(Modify Section 2.2.2, Types ) ::
++
+add *OpTypeAccelerationStructureNV* to list of opaque types
+
+
+(Modify Section 3.21, Builtin, adding rows to the Builtin table) ::
++
+--
+[cols="3^,1,6^,4*2",options="header",width = "100%"]
+|====
+2+^.^| Decoration | <<Capability,Enabling Capabilities>> 2+<.^| Extra Operands
+| 5337 | *HitMicroTriangleVertexPositionsNV* +
+The vertex positions for a hit with a microtriangle for the ray being traced.
+
+Allowed in *AnyHitKHR* and *ClosestHitKHR* stages.
+
+
+|*RayTracingDisplacementMicromapNV* 2+|
+| 5344 | *HitMicroTriangleVertexBarycentricsNV* +
+The barycentric coordinates for vertices of a microtriangle hit for the ray being traced relative to the base triangle's vertices.
+
+Allowed in *AnyHitKHR* and *ClosestHitKHR* stages.
+
+Refer to the Ray Tracing chapter of Vulkan API specification for more details.
+|*RayTracingDisplacementMicromapNV* 2+|
+| 5405 | *HitKindFrontFacingMicroTriangleNV* +
+A link-time constant that *HitKindKHR* will return when *OpTraceRayKHR* intersects a front facing microtriangle.
+
+Allowed in all stages.
+
+|*RayTracingDisplacementMicromapNV* 2+|
+| 5406 | *HitKindBackFacingMicroTriangleNV* +
+A link-time constant that *HitKindKHR* will return when *OpTraceRayKHR* intersects a back facing microtriangle.
+
+Allowed in all stages.
+|*RayTracingDisplacementMicromapNV* 2+|
+|====
+--
+
+(Modify Section 3.HK, Hit Kinds, adding rows to the table) ::
++
+--
+[[hit_kinds]]
+*3.HK, Hit Kinds*
+
+Values returned in the variable decorated as *HitKindKHR* from built-in
+intersections with triangle geometry.
+
+[cols="2,12,15",options="header",width = "100%"]
+|====
+Value^.^| Hit Kind | Enabling Capabilities
+| builtin | *HitKindFrontFacingMicroTriangleNV* +
+The intersection was with front-facing displacement micromap geometry.
+| *RayTracingDisplacementMicromapNV*
+| builtin | *HitKindBackFacingMicroTriangleKHR* +
+The intersection was with back-facing displacement micromap geometry.
+| *RayTracingDisplacementMicromapNV*
+|====
+--
+
+(Modify Section 3.31, Capability, adding a row to the Capability table) ::
++
+--
+[cols="3^.^,6,25,15^,15",options="header",width = "100%"]
+|====
+2+^.^| Capability|Enabled by Extension
```

Comment:
This table has columns `Capability` and `Enabled by Extension`, but usually those columns are `Capability` and `Implicitly Declares`. The headers for this extensions seem to imply that these capabilities would declare the `Shader` capability (which is unsurprising). I'd suggest modifying this table to match that. These capabilities are introduced by this extension so it doesn't add information to say they are enabled by it too.